src/main/scala/nl/knaw/dans/easy/ingest/Conf.scala

```diff
@@ -29,28 +32,30 @@ class Conf(args: Seq[String], props: PropertiesConfiguration) extends ScallopCon
                 |
                 |Usage:
                 |
-                | $printedName [-u <user> -p <password>] [-f <fcrepo-server>][-i] \\
+                | $printedName [-u <user> -p <password>] [-f <fcrepo-server>] [-i] \\
                 | $indent_____ [<staged-digital-object>... | <staged-digital-object-set>]
                 |
                 |Options:
                 |""".stripMargin)
+  private val s = " provided, please check {{app.home}}/cfg/application.properties"
```

Comment:
I don't think this message suffix really adds something useful. If you must add it, just repeat it in the strings below, and don't make the code harder to read by putting it into a variable 10 lines away.